# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/ufw/lock.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

diff -urN ufw-0.36.1.orig/src/ufw ufw-0.36.1/src/ufw
--- ufw-0.36.1.orig/src/ufw	2022-07-16 23:19:27.836807074 +0200
+++ ufw-0.36.1/src/ufw	2022-07-16 23:20:05.998806857 +0200
@@ -120,7 +120,7 @@
         raise
 
     if datadir is None:
-        lockfile = '/run/ufw.lock'
+        lockfile = '/var/run/ufw.lock'
         if os.getuid() != 0 or 'TESTSTATE' in os.environ:
             lockfile = os.path.join(ufw.common.state_dir, 'ufw.lock')
     else:
diff -urN ufw-0.36.1.orig/src/util.py ufw-0.36.1/src/util.py
--- ufw-0.36.1.orig/src/util.py	2022-07-16 23:19:27.836807074 +0200
+++ ufw-0.36.1/src/util.py	2022-07-16 23:20:22.261806765 +0200
@@ -1087,7 +1087,7 @@
     return binascii.unhexlify(h).decode('utf-8')
 
 
-def create_lock(lockfile='/run/ufw.lock', dryrun=False):
+def create_lock(lockfile='/var/run/ufw.lock', dryrun=False):
     '''Create a blocking lockfile'''
     lock = None
     if not dryrun:
